IDE/MonoDevelop/MonoDevelop.MonoGame.Android/MonoDevelop.MonoGame.Android.csproj

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProductVersion>8.0.30703</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{FB8E0F6F-522F-4C22-A2F2-1097EB549FE5}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>MonoDevelop.MonoGame.Android</RootNamespace>
+    <AssemblyName>MonoDevelop.MonoGame.Android</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>..\bin\Debug</OutputPath>
+    <DefineConstants>DEBUG;</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <ConsolePause>false</ConsolePause>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>full</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>..\bin\Release</OutputPath>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <ConsolePause>false</ConsolePause>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="MonoDevelop.Ide">
+      <HintPath>..\..\..\..\..\..\..\..\..\Applications\Xamarin Studio.app\Contents\Resources\lib\monodevelop\bin\MonoDevelop.Ide.dll</HintPath>
```

Comment:
No. Its constantly updated.

Also looks like some recent changes to the pipeline app on Linux have completely broken it on mac. We shouldn't be including .so files from a Linux box for the mac deployment. I get a DLL not found error now , its looking for libc.dylib for some reason :-/

-----Original Message-----
From: "Harry" <notifications@github.com>
Sent: ‎04/‎04/‎2015 20:38
To: "mono/MonoGame" <MonoGame@noreply.github.com>
Cc: "Dean Ellis" <dellis1972@googlemail.com>
Subject: Re: [MonoGame] Rework of Xamarin Studio/MonoDevelop Addin (#3361)

In IDE/MonoDevelop/MonoDevelop.MonoGame.Android/MonoDevelop.MonoGame.Android.csproj:
> +    <ErrorReport>prompt</ErrorReport>
> +    <WarningLevel>4</WarningLevel>
> +    <ConsolePause>false</ConsolePause>
> +  </PropertyGroup>
> +  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
> +    <DebugType>full</DebugType>
> +    <Optimize>true</Optimize>
> +    <OutputPath>..\bin\Release</OutputPath>
> +    <ErrorReport>prompt</ErrorReport>
> +    <WarningLevel>4</WarningLevel>
> +    <ConsolePause>false</ConsolePause>
> +  </PropertyGroup>
> +  <ItemGroup>
> +    <Reference Include="System" />
> +    <Reference Include="MonoDevelop.Ide">
> +      <HintPath>..\..\..\..\..\..\..\..\..\Applications\Xamarin Studio.app\Contents\Resources\lib\monodevelop\bin\MonoDevelop.Ide.dll</HintPath>
Wouldn't it be easier to include MonoDevelop.Ide.dll in MonoGame Dependencies?
—
Reply to this email directly or view it on GitHub.